scripts/modules_installer.py

```diff
@@ -18,38 +18,34 @@
  ***************************************************************************/
 """
 
-
-
 import subprocess
 import sys
 
-
 packages = sys.argv[1].split(',') if len(sys.argv) >= 2 else []
 
 # Adding the dependencies python modules in
 # package list in order to install via pip module
 
-    
-if not packages:
-    packages = [ 
-                'SQLAlchemy',
-                'SQLAlchemy-Utils',
-                'reportlab',
-                'networkx',
-                'matplotlib',
-                'PypeR',
-                'graphviz==0.8.3',
-                'pysftp',
-                'xlsxwriter',
-				'pandas'
-                ]
-
-
-
 
+if not packages:
+    packages = [
+        'SQLAlchemy',
+        'SQLAlchemy-Utils',
+        'reportlab',
+        'networkx',
+        'matplotlib',
+        'PypeR',
+        'graphviz==0.8.3',
+        'pysftp',
+        'xlsxwriter',
+        'pandas'
+    ]
+python_path = sys.exec_prefix
+python_version = sys.version[:3]
+cmd = '{}/bin/python{}'.format(python_path, python_version)
```

Comment:
if not packages:
    packages = [
        'SQLAlchemy',
        'SQLAlchemy-Utils',
        'reportlab',
        'networkx',
        'matplotlib',
        'PypeR',
        'graphviz==0.8.3',
        'pysftp',
        'xlsxwriter',
        'pandas'
    ]
python_path = sys.exec_prefix
python_version = sys.version[:3]

if platform.system()=='Windows':
    cmd = '{}/python'.format(python_path)

    # install pip if it is not found
    subprocess.check_call([cmd, '-m', 'ensurepip'], shell=False)

    for p in packages:
        subprocess.check_call([cmd, '-m', 'pip', 'install', '--upgrade', p], shell=False)
elif platform.system()=='Darwin':
    cmd = '{}\bin\python{}'.format(python_path,python_version)

    # install pip if it is not found
    subprocess.check_call([cmd, '-m', 'ensurepip'], shell=False)

    for p in packages:
        subprocess.check_call([cmd, '-m', 'pip', 'install', '--upgrade', p], shell=False)
        
elif platform.system()=='Linux':
    cmd = '{}\bin\python{}'.format(python_path,python_version)

    # install pip if it is not found
    subprocess.check_call([cmd, '-m', 'ensurepip'], shell=False)

    for p in packages:
        subprocess.check_call([cmd, '-m', 'pip', 'install', '--upgrade', p], shell=False)

Reply:
ok, ho aggiornato la PR per tenere in considerazione entrambe le piattaforme, dai un'occhiata.